.goreleaser.yml

```diff
@@ -0,0 +1,73 @@
+# This is an example goreleaser.yaml file with some defaults.
```

Comment:
👀 not an example anymore 😄

Reply:
Haha, true. I based this from https://github.com/hashicorp/packer-plugin-scaffolding/blob/24396fa085ccf54c056ab3a9831a939b58e03997/.goreleaser.yml. Should I removed all comments or just this line?

Reply:
It probably makes sense to remove comments that only apply to the example.  Thanks!